tests/Keboola/BaseFunctionalTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Tests;
+
+use Keboola\DbWriter\Test\BaseTest;
+use Symfony\Component\Process\Process;
+
+class BaseFunctionalTest extends BaseTest
+{
+    /** @var string */
+    protected $rootDir = __DIR__ . '/../../';
+
+    public function testRun(): void
+    {
+        $command = 'php ' . $this->rootDir . 'run.php --data=' . $this->rootDir . 'tests/data/functional 2>&1';
+        $process = new Process($command);
+        $process->run();
+        $this->assertEquals(0, $process->getExitCode());
```

Comment:
please put here `$prossess->getErrorOutput()` as the 3rd argument so that the log will show the error when it fails.

Reply:
👍